Mail/Views/Thread List/ThreadListCell.swift

```diff
@@ -33,10 +33,8 @@ struct ThreadListCell: View {
 
     @State private var shouldNavigateToThreadList = false
 
-    private var cellColor: Color {
-        return viewModel.selectedThread == thread
-            ? MailResourcesAsset.backgroundCardSelectedColor.swiftUiColor
-            : MailResourcesAsset.backgroundColor.swiftUiColor
+    private var selectedThreadBackground: Bool {
+        return !multipleSelectionViewModel.isEnabled && (viewModel.selectedThread?.uid == thread.uid)
```

Comment:
Why are we not using `viewModel.selectedThread == thread` as before instead of uid ?

Reply:
Because it compares the entire thread (seen value too)